include/onnxruntime/core/framework/tensor.h

```diff
@@ -69,11 +69,8 @@ class Tensor final {
 
   ~Tensor();
 
-  /**
-     Copy constructor and assign op will just pass the shape and memory
-     reference to another tensor. Not deep clone/copy.
-  */
-  Tensor(const Tensor& src);
+  Tensor(const Tensor& src) = delete;
```

Comment:
use the project macros - ORT_DISALLOW_COPY_AND_ASSIGNMENT

Reply:
Here is special. Because we have an implementation of operator= below. For me, I feel 'Tensor(const Tensor& src) = delete;' is more clear to see the difference.

Reply:
We have move assignment and move copy construct but not basic assignment, so ORT_DISALLOW_COPY_AND_ASSIGNMENT would actually do what you want. Otherwise you should also explicitly delete 'Tensor& operator=(Tensor& other)' as well

Reply:
The ORT_DISALLOW_ASSIGNMENT is doing the exact same thing as line 73 that you added.

Reply:
> We have move assignment and move copy construct but not basic assignment, so ORT_DISALLOW_COPY_AND_ASSIGNMENT would actually do what you want. Otherwise you should also explicitly delete 'Tensor& operator=(Tensor& other)' as well

Yes? I deleted it?